translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+
+#: pkg/kubectl/cmd/top_pod.go:80
+msgid "Display Resource (CPU/Memory) usage of pods"
+msgstr "Mostra a utlização de recursos (CPU/Memory) nos pode"
+
+#: pkg/kubectl/cmd/top.go:44
+msgid "Display Resource (CPU/Memory) usage."
+msgstr "Mostra a utlização de recursos (CPU/Memory)."
+
+#: pkg/kubectl/cmd/clusterinfo.go:51
+msgid "Display cluster info"
+msgstr "Mostra as informações do cluster"
+
+#: pkg/kubectl/cmd/config/get_clusters.go:41
+msgid "Display clusters defined in the kubeconfig"
+msgstr "Mostra os clusters definidos no kubeconfig"
+
+#: pkg/kubectl/cmd/config/view.go:67
+msgid "Display merged kubeconfig settings or a specified kubeconfig file"
+msgstr ""
+"Mostra a configuração do kubeconfig mescladas ou um arquivo kubeconfig "
+"especificado"
+
+#: pkg/kubectl/cmd/get.go:111
+msgid "Display one or many resources"
+msgstr "Mostra um ou mais recursos"
+
+#: pkg/kubectl/cmd/config/current_context.go:49
+msgid "Displays the current-context"
+msgstr "Mostra o contexto corrente"
+
+#: pkg/kubectl/cmd/explain.go:51
+msgid "Documentation of resources"
+msgstr "Documentaçao dos recursos"
+
+#: pkg/kubectl/cmd/drain.go:178
+msgid "Drain node in preparation for maintenance"
+msgstr "Drenar o nome para preparação de manutenção"
+
+#: pkg/kubectl/cmd/clusterinfo_dump.go:39
+msgid "Dump lots of relevant info for debugging and diagnosis"
+msgstr ""
+"Realiza o dump de muitas informações relevantes para debugging and "
+"diagnósticos"
+
+#: pkg/kubectl/cmd/edit.go:110
+msgid "Edit a resource on the server"
+msgstr "Edita um recurso no servidor"
+
+#: pkg/kubectl/cmd/create_secret.go:160
+msgid "Email for Docker registry"
+msgstr "Email para o Docker registry"
+
+#: pkg/kubectl/cmd/exec.go:69
+msgid "Execute a command in a container"
+msgstr "Executa um comando em um container"
+
+#: pkg/kubectl/cmd/rollingupdate.go:103
+msgid ""
+"Explicit policy for when to pull container images. Required when --image is "
+"same as existing image, ignored otherwise."
+msgstr ""
+"Política explícita para quando extrair imagens de contêiner. Obrigatório "
+"quando --image for igual à imagem existente, caso contrário, será ignorado."
+
+#: pkg/kubectl/cmd/portforward.go:76
+msgid "Forward one or more local ports to a pod"
+msgstr "Encaminhar uma ou mais portas locais para um pod"
+
+#: pkg/kubectl/cmd/help.go:37
+msgid "Help about any command"
+msgstr "Ajuda sobre qualquer comando"
+
+#: pkg/kubectl/cmd/expose.go:103
+msgid ""
+"IP to assign to the Load Balancer. If empty, an ephemeral IP will be created "
+"and used (cloud-provider specific)."
+msgstr ""
+"IP para ser alocado no Load Balancer. Se vazio, um IP efêmero será criado e "
+"utilizado (específico para cada provedor cloud)."
+
+#: pkg/kubectl/cmd/expose.go:112
+msgid ""
+"If non-empty, set the session affinity for the service to this; legal "
+"values: 'None', 'ClientIP'"
+msgstr ""
+"Se não vazio, configura a session affinity para o serviço; valores válidos: "
```

Comment:
configura a afinidade de sessão (session affinity have a good meaning in portuguese when translated)